meta-sokol-flex-support/recipes-core/meta/codebench-makefile.bb

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: MIT
+# ---------------------------------------------------------------------------------------------------------------------
+
+SUMMARY = "ADE Makefile Fragment for CodeBench"
```

Comment:
This occurance of ADE left.

Reply:
Fixed. Also updated to remove a remnant ref in sokol-flex.conf and to drop zip from setup-ubuntu.